qsam/readme.txt

```diff
@@ -0,0 +1,28 @@
+This qsam directory contains regression tests for qsam get and put
```

Comment:
Given we are using GitHub, the better format to use for files like this is markdown rather than text.
GitHub will automatically render README.MD files when viewing MD files. Recommendation only.

Reply:
I have changed this to CHANGELOG.md in the automated build PR so I would just leave as is for this change.